test/parallel/test-regression-object-prototype.js

```diff
@@ -1,7 +0,0 @@
-'use strict';
```

Comment:
It looks to be ancient and related to c-ares: https://github.com/nodejs/node/commit/cc9d5ab546e92ff9e56e4355e0cf0e557221bbab

I'm think fine with removing it.